pyorbital/tests/test_geoloc.py

```diff
@@ -27,7 +27,7 @@
 import numpy as np
 
```

Comment:
```suggestion
import pytest
```
Then the erroneous `self.assertRaises` below can be replaced